### ISSUE TITLE
Flashback (REminiscence Core) won't launch by command line

### ISSUE THREAD

**Issue by schmurtzm** [jimgraygit/Onion#346]
Hi, I try to create a "launch.sh" to run Flashback Port ([libretro documentation](https://docs.libretro.com/library/reminiscence/)) directly from the new PORTS menu
Result for the moment Retroarch is launched with a blackscreen. If I go in retroarch menu and start flashback manually it works.
I don't know if I 've made an error in my launch.sh : 

Seems OK to me... a bug in the core ? 🤔 

```bash
#!/bin/sh
echo $0 $*
progdir=`dirname "$0"`
homedir=`dirname "$1"`



if [ -f "/mnt/SDCARD/Emu/PORTS/Binaries/Flashback.port/FILE_HERE/level1.lev" ]; then
	# Timer initialisation
	cd /mnt/SDCARD/App/PlayActivity
	./playActivity "init"

	cd /mnt/SDCARD/RetroArch/
	HOME=/mnt/SDCARD/RetroArch/ /mnt/SDCARD/RetroArch/retroarch -v -L /mnt/SDCARD/RetroArch/.retroarch/cores/reminiscence_libretro.so "/mnt/SDCARD/Emu/PORTS/Binaries/Flashback.port/FILE_HERE/level1.lev"

	# Timer registration
	cd /mnt/SDCARD/App/PlayActivity
	./playActivity "Flashback"
else
	cd "/mnt/SDCARD/Emu/PORTS/Binaries/missingFile"
	./infoPanel
fi
```

Here my files (without games content) : 
https://dl.dropbox.com/s/v0cdwqdoirp30kc/flashback.zip

**Comment by Olywa** [comment]
I had a quick look at your script and I don't see anything immediately wrong (maybe someone with more knowledge says otherwise) but I also don't think it can be a bug (if the core works launching manually then there is no reason why the ports launch script would not work (as long as the script (paths) and your actual folders and filenames are all correct).

The only thing I can suggest is to check the case of your folders/files to be sure they are matched in the script (i.e. is 'level1.lev' named exactly this (with no capitals).

**Comment by Olywa** [comment]
On a personal note, I would suggest that for some games, setting up ports seems a little counterproductive when these games (i.e. Flashback) exist already on other systems (Megadrive, Snes, Amiga), accepting that (in some cases) with a lesser soundtrack ;)

**Comment by schmurtzm** [comment]
> On a personal note, I would suggest that for some games, setting up ports seems a little counterproductive when these games (i.e. Flashback) exist already on other systems (Megadrive, Snes, Amiga), accepting that (in some cases) with a lesser soundtrack ;)

It's a kind of personal challenge to test all the cores available 😉 ! I like to make things work (may be more than playing 😄).
(May be I will have a similar question with mrboom and others later 😅).
For example Quake 1 ports and doom ports offer a better experience than on an emulator I think (Quake 1 fluidity is impressive on the Miyoo Mini).

I've double checked my path, I don't see the problem, may be someone will have a solution but as you said there are alternatives to play this game.

**Comment by Sichroteph** [comment]
Could you share us your work on the other ports if you moved them to the new space ? Really interested by quake :)

**Comment by dwmccoy** [comment]
I know that the blank screen when starting this game is a known issue.  If you open Retroarch at the blank screen and choose "Unload Content", then go to "History", select "Level1", then choose "Run", the game will display and work correctly.  Research on this topic will reveal this is the fix for those who experience this issue on other emulation platforms.

**Comment by schmurtzm** [comment]
> I know that the blank screen when starting this game is a known issue. If you open Retroarch at the blank screen and choose "Unload Content", then go to "History", select "Level1", then choose "Run", the game will display and work correctly. Research on this topic will reveal this is the fix for those who experience this issue on other emulation platforms.

Yes, it works when you do it manually. I really suspect a bug in the core because the command "restart" doesn't work neither : as you said you really have to close the content and launch again... 

I will try on PC to see if the core has the same behavior...
